core/domain/classifier_services.py

```diff
@@ -213,12 +306,6 @@ def create_classifier(job_id, classifier_data):
         classifier.state_name, classifier.algorithm_id,
         classifier.classifier_data, classifier.data_schema_version)
 
-    # Create the mapping from <exp_id,exp_version,state_name>
-    # to classifier_id.
-    create_classifier_exploration_mapping(
-        classifier.exp_id, classifier.exp_version_when_created,
-        classifier.state_name, classifier_id)
-
     return classifier_id
```

Comment:
@seanlip We hadn't made a call on that yet. That would involve major upheaval everywhere and I think its better to loop in the rest of the team. @anmolshkl @prasanna08 Sean suggested that we have the classifier_data as a job attributes to avoid extra GET ops. If we do that, we could actually remove ClassifierData model completely. What are your thoughts?